cms/env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Hva er denne for?

Reply:
Den er fra dokumentasjonen. Vi har ikke bruk for den enda, men vi kan bruke det til å gjøre env-variabler typesafe.

Reply:
https://vitejs.dev/guide/env-and-mode.html